names.sql

```diff
@@ -0,0 +1,58 @@
+-- Búa til the name table ef hún er ekki þegar til
+CREATE TABLE IF NOT EXISTS names (
+    name TEXT NOT NULL,
+    year INTEGER NOT NULL,
+    frequency INTEGER NOT NULL,
+    type TEXT NOT NULL,
+    PRIMARY KEY (name, year, type)
+);
+
+--  Búa til temborary töflu frá cvs
+CREATE TABLE IF NOT EXISTS temp (
+    name TEXT NOT NULL,
+    year INTEGER NOT NULL,
+    frequency INTEGER NOT NULL
+);
+
+-- Setja eignafn skránna inn í nafna töfluna
+.mode csv
+.import data/first_names_freq.csv temp
```

Comment:
```suggestion 
.import --skip 1 data/first_names_freq.csv temp
```
vantar að sleppa að lesa inn header línuna